components/python/jsonrpclib/Makefile

```diff
@@ -28,13 +28,13 @@ BUILD_STYLE=setup.py
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		jsonrpclib
-COMPONENT_VERSION=	0.4.0
-COMPONENT_REVISION=	4
+COMPONENT_VERSION=	0.4.3
+COMPONENT_REVISION=	5
```

Comment:
If you change the COMPONENT_VERSION you should reset the COMPONENT_REVISION, eg. by removing it.